src/OpenApi/test/Services/OpenApiDocumentServiceTests.Info.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.AspNetCore.OpenApi;
+using Microsoft.Extensions.Hosting.Internal;
+using Microsoft.Extensions.Options;
+using Moq;
+
+public partial class OpenApiDocumentServiceTests
+{
+    [Fact]
+    public void GetOpenApiInfo_RespectsHostEnvironmentName()
+    {
+        // Arrange
+        var hostEnvironment = new HostingEnvironment
+        {
+            ApplicationName = "TestApplication"
+        };
+        var docService = new OpenApiDocumentService(
+            "v1",
```

Comment:
Would it be worthwhile to test with a document name other than the default?

Reply:
Yep -- doesn't hurt to add another test here.